src/util.coffee

```diff
@@ -32,3 +32,25 @@
                                       node?.tag is 'tag:yaml.org,2002:map'
 
 @NON_PRINTABLE = /[^\x09\x0A\x0D\x20-\x7E\x85\xA0-\uD7FF\uE000-\uFFFD]/
+
+@stringifyJSONCircularDepth = (json, depth = 1000) ->
```

Comment:
With this approach you are letting the users to choose it.
I mean, if this is the default approach we can put `maxDepth=0` by default. But for `raml-mocker` library I think is better to have high depth in order to have more diversity of generated jsons.